### PR TITLE
[RSDK-7321] Rename AnalogReaders -> Analogs

### PR DESCRIPTION
**Pull request by randhid**
Still a few renames to go in resource manager and injects, feel free to pick those up if you have time today.

### CODE REVIEW

**Review by penguinland — APPROVED** [review]
This is a pretty major breaking change, which will cause Go SDK users' programs to stop compiling. Is there a Jira ticket associated with this? I believe it was approved as part of [this scope doc](https://viam.atlassian.net/browse/RSDK-6933), but we should probably track this change more closely.

(Admittedly, very few people are using analog readers and it's possible that none of them are using the Go SDK so the change won't actually affect anyone. but I think it's something we should be careful about.)

Beyond that documentation of the work done, LGTM!

**Comment by penguinland** [comment]
This is also going to require documentation updates, which will be easier if there's a Jira ticket attached.

**Comment by randhid** [comment]
> This is a pretty major breaking change, which will cause Go SDK users' programs to stop compiling. Is there a Jira ticket associated with this? I believe it was approved as part of [this scope doc](https://viam.atlassian.net/browse/RSDK-6933), but we should probably track this change more closely.
> 
> (Admittedly, very few people are using analog readers and it's possible that none of them are using the Go SDK so the change won't actually affect anyone. but I think it's something we should be careful about.)
> 
> Beyond that documentation of the work done, LGTM!

The board docs are already not super useful, and not documenting the functions that go over the wire: https://docs.viam.com/components/board/#readanalogreader. I'm comfortable waiting until the Golang API is entirely changed; I expect us to take up some of that work.

If you'd like - make a jira ticket and I'll attach it.

**Review by oliviamiller — APPROVED** [review]

File: web/frontend/src/components/board/index.svelte [hunk 1]
```diff
@@ -95,7 +95,7 @@ const writeAnalog = async () => {
 
 const readAnalog = async () => {
   try {
-    const value = await boardClient.readAnalogReader(analogPinName);
+    const value = await boardClient.readAnalogReadcd .er(analogPinName);
```
Comment (ethanlook) [inline web/frontend/src/components/board/index.svelte:98]: This doesn't look right - what's going on here?

Comment (randhid) [reply web/frontend/src/components/board/index.svelte]: an unfortunate typo when I was looking for mentions of analog reader in rdk.

Comment (randhid) [reply web/frontend/src/components/board/index.svelte]: reverted :)

Comment (ethanlook) [reply web/frontend/src/components/board/index.svelte]: Thanks! Removing myself because there are no more RC changes :)

**Pull request merged**